packages/okta-react/test/e2e/harness/package.json

```diff
@@ -26,7 +26,7 @@
     "build": "CLIENT_ID=${SPA_CLIENT_ID:=$CLIENT_ID} && REACT_APP_ISSUER=$ISSUER REACT_APP_CLIENT_ID=$CLIENT_ID PORT=8080 OKTA_TESTING_DISABLEHTTPSCHECK=$OKTA_TESTING_DISABLEHTTPSCHECK react-scripts build",
     "build:test": "rimraf e2e/dist && babel e2e/ -d e2e/dist",
     "kill:port": "kill -9 $(lsof -t -i:8080 -sTCP:LISTEN) || true",
-    "pretest": "yarn kill:port && ./node_modules/.bin/webdriver-manager update --versions.chrome 2.45  --gecko false && yarn build:test",
+    "pretest": "yarn kill:port && ../../../../../scripts/updateSeDrivers.sh && yarn build:test",
```

Comment:
We have the ongoing discussion about making this more cross-platform at some point, but I don't see a problem with this for now - it doesn't make it LESS cross-platform.